web/src/components/ToggleButton.tsx

```diff
@@ -39,10 +39,15 @@ export default function ToggleButton({
     }
   };
   return (
-    <div className="z-10 flex h-9 rounded-full bg-zinc-100  px-[5px] py-1  drop-shadow-lg dark:bg-gray-900">
+    <div
+      className={twMerge(
+        'z-10 flex h-9 rounded-full bg-gray-200/80 px-1  shadow    dark:bg-gray-900',
+        transparentBackground ? 'bg-gray-200/80 backdrop-blur-sm' : 'bg-gray-200'
+      )}
+    >
       <ToggleGroupRoot
         className={
-          'flex-start flex h-[26px] flex-grow flex-row items-center justify-between self-center rounded-full bg-gray-100 shadow-inner dark:bg-gray-700'
+          'flex-start flex flex-grow flex-row items-center justify-between self-center  dark:bg-gray-700'
```

Comment:
I think you still need the rounded-full class here, I think it's removal is what made this happen: 😅
![image](https://user-images.githubusercontent.com/30777521/225571256-7ff41b34-15a1-45d4-a7cc-df377388d13c.png)

Reply:
Nice spotting :)